input/fsh/au-erequesting-servicerequest.fsh

```diff
@@ -188,6 +188,19 @@ Description: "This profile sets minimum expectations for a ServiceRequest resour
 * reasonCode ^extension[http://hl7.org/fhir/StructureDefinition/obligation][4].extension[code].valueCode = #SHALL:no-error
 * reasonCode ^extension[http://hl7.org/fhir/StructureDefinition/obligation][4].extension[actor][0].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-patientaccess"
 
+* priority MS
+* priority from http://hl7.org/fhir/ValueSet/request-priority (required)
```

Comment:
This is already a required binding in the core FHIR ServiceRequest profile, no need to replicate it here. Please remove.

Reply:
Removed, thanks.